### ISSUE TITLE
Icons on FlxUIButtons are stamped on top of each other

### ISSUE THREAD

**Issue by happinesssam** [HaxeFlixel/flixel-ui#200]
I recently updated from 1.0.2 to 2.2.0.
I've been creating some FlxUIButtons dynamically and previously I was stamping them from another FlxSprite that has all the frames. On updating it seemed that the icons were all getting stamped on top of each other.

After scratching my head and looking into it I found what I think is the issue - on line 192 of FlxUIButton it loads the new graphic, but the new bitmap isn't unique and the keys will be the same since the buttons and icon all share the same assets. When I amended it so that the new bitmap was unique the issue was resolved and the icons displayed correctly. I am not sure if there would be any problems down the line from doing this.

In summary, currently line 192 of FlxUIButton is
loadGraphic(newGraphic, true, Std.int(width), Std.int(height));
 and my icons look like this:
![image](https://cloud.githubusercontent.com/assets/4408520/19370314/17a8a514-91a3-11e6-9d02-4c865596ad75.png)

If I amend that line to
loadGraphic(newGraphic, true, Std.int(width), Std.int(height), true);
my icons look like this:
![image](https://cloud.githubusercontent.com/assets/4408520/19370344/4c231888-91a3-11e6-84d1-80d889f78313.png)
